### DISCUSSION TITLE
Seq Upgrade from 2022.1.8000-pre to 2024.2.11456

### DISCUSSION THREAD

**Discussion by specialagentsoftware** [datalust/seq-tickets#2211] [General]
Hello. 

We have a couple of environments installed here. Dev is a single instance and Prod is a cluster install all Windows Server. We update Dev more frequently than we do Prod so Dev had pretty regular staggered updates between the two versions mentioned in the title. Prod however is currently on 2022.1.8000-pre and we are going to update it to 2024.2.11456. 

I was looking at the upgrade and the upgrade cluster instructions. Everything there is pretty straight forward. I just had a question. When I look at the disk structure of 2022.1.8000-pre that is on prod, there are configuration files at C:/ProgramData/Seq. If I look at the install of 2024.2.11456 in Dev I see that all of the config that was in C:/ProgramData/Seq is stored in the seq installation directory. I am specifically looking at the config files, etc. At the same time of our upgrade to the Prod cluster we are going to add an EBS volume and move the Seq install to its own disk like we did in Dev. 

When I ran the in-place upgrade on a test machine going from 2022.1.8000-pre to 2024.2.11456 I did not see any changes to where the files were stored. I was wondering if that should have happened or if that happened during an upgrade that was pushed out between those two releases that I should install before I install 2024.2.11456 on the Prod machine. 

Thanks, I hope that made sense.

**Comment by nblumhardt** [comment]
Hi Jeremy,

None of these upgrades should change the instance's storage location; my guess is that during post-install configuration in Dev, someone set the storage path (in the post-install wizard) to the alternative location.

I'd recommend doing the upgrade in-place, without switching to the new disk, and then afterwards (once you've verified the upgrade has succeeded) copy/move `C:\ProgramData\Seq` to the new location.

To perform the move,

1. Stop the service (`seq service stop`)
2. Move the folder
3. Run Seq Service Administration from the Windows Start menu, and choose the new path to the`Seq/` folder before clicking through to the end of the wizard (which will also restart the service)

Always best to ensure you have backups of anything vital first, just to be on the safe side, although the process should be very straightforward.
